ui/webui/test/check-basic

```diff
@@ -48,6 +48,34 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         # Ensure that the 'actual' UI process is running/
         self.assertIn("/usr/libexec/webui-desktop", m.execute("ps aux"))
 
+    def testSidebarNavigation(self):
+        b = self.browser
+        m = self.machine
+        i = Installer(b, m)
+
+        i.open()
+
+        i.check_prerelease_info()
+
+        i.click_step_on_sidebar()
```

Comment:
At first, I wrote review comments how the "no step" is not used, then found it. Maybe this call could have a comment that it's testing clicking the same step?

Reply:
Updated